jihyeon/controllers/userController.js

```diff
@@ -16,6 +16,19 @@ const signUp = async (req, res) => {
   }
 };
 
+const getUserByEmail = async (req, res) => {
```

Comment:
```suggestion
const getUserByEmail = async (req, res) => {
```
컨트롤러 함수 이름은 login 혹은 signIn으로 바꿔주세요.